website/source/guides/operations/autounseal-aws-kms.html.md

```diff
@@ -0,0 +1,282 @@
+---
+layout: "guides"
+page_title: "Vault Auto-unseal using AWS KMS - Guides"
+sidebar_current: "guides-autounseal-aws-kms"
+description: |-
+  In this guide, we'll show an example of how to use Terraform to provision an
+  instance that can utilize an encryption key from AWS Key Management Services
+  to unseal Vault.
+---
+
+
+# Vault Auto Unseal using AWS Key Management Service
+
+~> **Enterprise Only:** Vault replication feature is a part of _Vault Enterprise_.
```

Comment:
Did you mean "auto unsealing" instead of "replication"?

Reply:
Ops.  Yes!  Thank you for catching.  My bad from copying & pasting.